CouchBaseStorage/BaseJSONStorageProvider.cs

```diff
@@ -51,6 +53,7 @@ protected BaseJSONStorageProvider()
         public virtual Task Init(string name, IProviderRuntime providerRuntime, IProviderConfiguration config)
         {
             this.Name = name;
+            this.jsonSerializerSettings = OrleansJsonSerializer.GetDefaultSerializerSettings();

```

Comment:
Good, this is the correct place to put this, since this call will change slightly when you update to v1.5 with non-static `SerializationManager` (don't worry, it's a tiny change)